src/modules/blockchainApplication/manage/hooks/useGetMergedApplication.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable max-statements */
+import { useEffect, useMemo } from 'react';
+import { metaDataClient } from 'src/utils/api/client';
+import networks from 'src/modules/network/configuration/networks';
+import { useBlockchainApplicationExplore } from '../../explore/hooks/queries/useBlockchainApplicationExplore';
+import { useBlockchainApplicationMeta } from './queries/useBlockchainApplicationMeta';
+
+const extractAppMetaDataToChainIds = (applicationsMetaData) =>
+  applicationsMetaData.map(({ chainID }) => chainID).join(',');
+
+export const useGetMergedApplication = ({ params = {}, networkName, isEnabled } = {}) => {
+  const {
+    data: { data: applicationsMetadata = [] } = {},
+    isFetched: isFetchedAppsMetaData,
+    isLoading: isLoadingAppsMetaData,
+    error: errorGettingAppsMetaData,
+    refetch: refetchAppsMetaData,
+  } = useBlockchainApplicationMeta({
+    config: { params: { chainID: params.chainID, network: networkName } },
+    client: metaDataClient,
+    options: { enabled: isEnabled },
+  });
+
+  const isBlockchainExporeEnabled =
```

Comment:
missing l in expLore
`isBlockchainExploreEnabled`

Reply:
would be deprecated by https://github.com/LiskHQ/lisk-desktop/pull/4859/#issue-1631188237